src/main/java/com/jvm_bloggers/kafka/message/NewIssuePublishedMessage.java

```diff
@@ -0,0 +1,14 @@
+package com.jvm_bloggers.kafka.message;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.Value;
+
+@KafkaMessage
+@Value
+public class NewIssuePublishedMessage {
+    @JsonProperty("issueNumber")
```

Comment:
Are these `@JsonProperty` annotations really required if the field names are equal to the annotation value?